chart/templates/ingresses.yaml

```
@@ -0,0 +1,52 @@
+{{- $ingress := (merge (dict "name" "ingress" "module" "dashboard") .) -}}
+{{- $backendService := (merge (dict "name" "backend-service" "module" "dashboard") .) -}}
+{{- $frontendService := (merge (dict "name" "frontend-service" "module" "dashboard") .) -}}
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "chart.prefixedName" $ingress }}
+  labels:
+    {{- include "chart.labels" $ingress | nindent 4}}
+  {{- if .Values.backend.ingress.annotations }}
+  annotations: 
+  {{ toYaml .Values.backend.ingress.annotations | indent 2}}
+  {{- end }}
+spec:
+  rules:
+  {{- if .Values.host }}
+  - host: {{ .Values.host }}
+    http:
+  {{- else  }}
+  - http:
+  {{- end }}
```

Given how the chart is built, it would probably better to throw an error is .Values.host is not set.